test/compilable/fieldwise.d

```diff
@@ -0,0 +1,11 @@
+/* REQUIRED_ARGS: -preview=dip1000 -preview=fieldwise
```

Comment:
If this change isn't guarded by `fieldwise`, do you still need the flag here?

Reply:
Yes, because fieldwise changes the behavior before it.